### PR TITLE
Test for sorting packages on the project show page

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

**Comment by bgeuken** [comment]
@ChrisBr Please review.

I added the scope so that we sort them case-insensitive.
The Pack3 change was made for testing sorting inherited packages case-insensitive

File: src/api/app/controllers/webui/project_controller.rb [hunk 1]
```diff
@@ -901,8 +901,9 @@ def set_maintained_project
 
   def load_project_info
     find_maintenance_infos
-    @ipackages = @project.linked_packages.map { |package| [ package.name, package.project.name ] }.sort
-    @packages = @project.packages.order(name: :desc).pluck(:name)
+    @ipackages = @project.linked_packages.map { |package| [ package.name, package.project.name ] }
```
Comment (ChrisBr) [inline src/api/app/controllers/webui/project_controller.rb]: The linked_packages are already sorted?

**Comment by ChrisBr** [comment]
LGTM

File: src/api/app/controllers/webui/project_controller.rb (hunk 1 above)
Comment (bgeuken) [inline src/api/app/controllers/webui/project_controller.rb]: It seems so. At least for that project. I would risk it and see if this is also true in OBS online

**Pull request merged**